docs/getting-started.md

```diff
@@ -8,8 +8,8 @@
 Argo Rollouts can be installed at a cluster or namespace level. 
 
 !!! important
-    Since the API server rejects CRD manifests with unknown fields, the Argo Rollout's CRDs do not work with **v1.13.x** 
-    because they have a new field from the v1.14. Adding the `--validate=false` to the commands below allows the cluster to 
+    Since the API server rejects CRD manifests with unknown fields, the Argo Rollout's CRDs do not work with **v1.14.x**
+    because they have a new field from the v1.15. Adding the `--validate=false` to the commands below allows the cluster to
```

Comment:
The way this is worded makes it seem we don't work with v1.14. Lets rephrase it as a workaround:

When installing Argo Rollouts on Kubernetes v1.14 or lower, the CRD manifests must be kubectl applied with the `--validate=false` option. This is caused by use of new CRD fields introduced in v1.15, which are rejected by default in lower API servers.

Reply:
in kubernetes v1.14 the Rollout Controller found this err ，does that mean the Argo Rollout's CRDs do not work with v1.14?

```
ERRO[2021-05-10T17:07:05+08:00] rollout syncHandler error: the server could not find the requested resource (put rollouts.argoproj.io apiserver-demo)  namespace=default rollout=rollout-demo
E0510 17:07:05.309393     421 controller.go:172] the server could not find the requested resource (put rollouts.argoproj.io rollout-demo)
```

this may be relevant...:https://github.com/operator-framework/operator-sdk/issues/1124#issuecomment-499165754